test/test_helper.exs

```diff
@@ -1 +1,5 @@
-ExUnit.start(capture_log: true, exclude: [long: true])
+excluded = [
+  long: true
+]
+
+ExUnit.start(capture_log: true, exclude: excluded)
```

Comment:
This seems unecessary complicated, but we can leave it that way. Also, I commented on previous PR but it wasn't applied: we can just tag tests with `@tag :long` (instead of `@tag long: true`)